packages/nimble-components/src/radio/styles.ts

```diff
@@ -30,6 +30,7 @@ export const styles = css`
     }
 
     .control {
+        position: relative;
```

Comment:
I cringe whenever we implement a change and don't have a way to auto test it. I guess we don't auto test any of our focus states so there's precedence for this, but it's worth a quick brainstorm to see if we could come up with something to improve matters.

One idea is a unit test that queries for an element in the shadow dom that has `position: relative` set on it and fails if it's not found. At least that way, if we refactor the component and unwittingly remove this, a test will fail and we'll pause to think about why.

Thoughts?

Reply:
Fair enough, I'm ok to skip a test in this case. Thanks for thinking about it.